addons/website_livechat/models/discuss_channel.py

```diff
@@ -24,17 +25,10 @@ def channel_pin(self, pinned=False):
         if self.livechat_active and not self.message_ids:
             self.sudo().unlink()
 
-    def _field_store_repr(self, field_name):
-        if field_name == "requested_by_operator":
-            return [
-                Store.Attr(
-                    "requested_by_operator",
-                    # sudo - res.users: can access operator's user even if he left the channel.
-                    lambda channel: channel.create_uid in channel.livechat_operator_id.sudo().user_ids,
-                    predicate=lambda channel: channel.livechat_visitor_id,
-                ),
-            ]
-        return super()._field_store_repr(field_name)
+    def _compute_requested_by_operator(self):
+        for channel in self:
+            # sudo - res.users: can access operator's user even if he left the channel.
+            channel.requested_by_operator = channel.create_uid in channel.livechat_operator_id.sudo().user_ids
```

Comment:
Now that we have history, it seems like the value should be computed based on history instead, in particular checking if the first member is an agent, if that is indeed the case in that flow?

And if not possible, then we might want to actually store the field and set it to true in the flow, rather than computing it?